CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v11.0.0
+
+**Deprecation Notice**
+v11.0.0 of the client follows the Kubernetes [deprecation policy](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew) and will
+be deprecated as and when Kubernetes version v1.15 gets deprecated.
```

Comment:
I didn't quite get "as and when k8s v1.15..."?

Reply:
I meant that whenever in future k8s v1.15 is deprecated, we will automatically drop support for Client version 11.0.0

Reply:
I think it would be clearer if you removed the "as and". You can also remove "version" since you have 'v1.15'.